Userland/Applications/Browser/BookmarksBarWidget.cpp

```diff
@@ -31,7 +31,7 @@ class BookmarkEditor final : public GUI::Dialog {
     C_OBJECT(BookmarkEditor)
 
 public:
-    static Vector<JsonValue>
+    static Optional<Vector<JsonValue>>
```

Comment:
Taking another look at `edit_bookmark` here, in lines 45-49 the original author's intention was clear: if the user did not press OK, an empty vector is returned. So I believe it's fine to leave the return type to be `Vector<JsonValue>` and check for  `.is_empty()` instead of `.has_value()`.